src/shapes/object.class.js

```diff
@@ -321,6 +321,13 @@
      */
     strokeDashArray:          null,
 
+    /**
+     * Line offset of an object's stroke (can be a number or a %)
```

Comment:
are we actually supporting % out of the box? how?

Reply:
no you're right it's only working on svg